image_provisioner/playbooks/roles/docker-config/tasks/main.yaml

```diff
@@ -1,4 +1,40 @@
 ---
+- name: set selinux to enforcing
+  selinux:
+    policy: targeted
+    state: enforcing
+
+- name: restart server
+  command: shutdown -r now "Reboot started"
+  async: 0
+  poll: 0
+  ignore_errors: true
+  when: docker_storage_driver == "overlay" or update_kernel is defined
```

Comment:
update_kernel should be bool type, then use more strict checks rather than just defined